libsnark/gadgetlib1/gadgets/hashes/anemoi/tests/anemoi_outputs.cpp

```diff
@@ -0,0 +1,93 @@
+/** @file
+ *****************************************************************************
+ * @author     This file is part of libff, developed by Clearmatics Ltd
+ *             (originally developed by SCIPR Lab) and contributors
+ *             (see AUTHORS).
+ * @copyright  MIT license (see LICENSE file)
+ *****************************************************************************/
+
+#include "libsnark/gadgetlib1/gadgets/hashes/anemoi/tests/anemoi_outputs.hpp"
+
+namespace libsnark
+{
+
+using ppT = libff::bls12_381_pp;
```

Comment:
Again, this kind of implies that we will only ever have bls12-381 values in this scope and therefore file.

Reply:
You could potentially move these into the function, if they are for convenience.

Reply:
Correct. Same problem as above. Fixed to explicitly state the type. Just to clarify: my intention here is to have one such "expected outputs" function for every curve for the sake of unit testing.